internal/server/evaluation/ofrep_bridge.go

```diff
@@ -0,0 +1,85 @@
+package evaluation
+
+import (
+	"context"
+	"errors"
+	"strconv"
+
+	"github.com/google/uuid"
+
+	"go.flipt.io/flipt/internal/server/ofrep"
+
+	rpcevaluation "go.flipt.io/flipt/rpc/flipt/evaluation"
+
+	fliptotel "go.flipt.io/flipt/internal/server/otel"
+	"go.flipt.io/flipt/internal/storage"
+	"go.flipt.io/flipt/rpc/flipt"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func (s *Server) OFREPEvaluationBridge(ctx context.Context, input ofrep.EvaluationBridgeInput) (ofrep.EvaluationBridgeOutput, error) {
+	flag, err := s.store.GetFlag(ctx, storage.NewResource(input.NamespaceKey, input.FlagKey))
+	if err != nil {
+		return ofrep.EvaluationBridgeOutput{}, err
+	}
+
+	span := trace.SpanFromContext(ctx)
+	span.SetAttributes(
+		fliptotel.AttributeNamespace.String(input.NamespaceKey),
+		fliptotel.AttributeFlag.String(input.FlagKey),
+		fliptotel.AttributeProviderName,
+	)
+
+	req := &rpcevaluation.EvaluationRequest{
+		NamespaceKey: input.NamespaceKey,
+		FlagKey:      input.FlagKey,
+		// OFREP does not have the idea of an `entity` and this is why we're generating one here.
+		// I might change in the future as the specification grows.
+		EntityId: uuid.NewString(),
```

Comment:
I have a quick question about this, @markphelps might be able to answer off the top of his head:
Do we do anything in the existing OF clients to support `EntityId`?

My current thought is that without being able to control EntityId via the client, we loose all the benefits of being able to make thresholds sticky to some identifier. The ability to make percentage rollouts sticky has always been quite important to me in the past. Usually at-least to some user or org level, as I rarely want a feature to flip-flop between on and off for a single user agents session.

I wonder if its worth us having a special context key that can be defined on the OF context which overrides this.

Doesn't have to be in this PR, but maybe we open an issue at-least. Could also be a part of this change though, it doesn't need to be complex, something like:
```go
const fliptOFEntityIdKey = "__fliptEntityId";

if eid, ok := input.Context[fliptOFEntityIdKey]; ok {
  req.EntityId = eid
}
```

Reply:
Yeah I was thinking of something similar. A well known/documented key in the context to map to entityID

I like your suggestion 

We could also ask in the OFREP slack if they have thought of this or have any suggestions themselves

Reply:
I guess we could add a new attribute named "metadata" or something similar, or even allow the user to pass custom headers on request time instead of client instantiation time! This should be an addition to OFREP spec

Reply:
i think just documenting a 'known' key in the context that they send for evaluation as @GeorgeMac suggested would be ideal instead of another header value. just my two cents though. i'd be interested to see what the OFREP peeps suggest

but its not blocking shipping of this PR i don't think, but we prob want to resolve it before we create a new release and announce OFREP support

Reply:
Just checked w the OFREP team. It's not documented currently but the field is called 'targetingKey' in the OFREP clients in the context. So we should map that field to entity id in Flipt server implementation 

https://cloud-native.slack.com/archives/C066A48LK35/p1721921651364699?thread_ts=1721915087.008769&channel=C066A48LK35&message_ts=1721921651.364699

I think they are going to add it to the openapi spec

Reply:
That's awesome that there is a standard!